cloud_functions/token_refresh/utils/task_utils.py

```
@@ -0,0 +1,114 @@
+
+    Args:
+        encrypted_token (str): The encrypted token in format "iv:encrypted"
+        key (str or bytes): The encryption key
+
+    Returns:
+        str: The decrypted token
+    """
+    # Split the IV and encrypted data
+    parts = encrypted_token.split(":")
+    if len(parts) != 2:
+        raise ValueError("Invalid encrypted token format")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)